lib/timetrap/cli.rb

```diff
@@ -443,7 +447,12 @@ def yesterday
     end
 
     def week
-      args['-s'] = Date.today.wday == 1 ? Date.today.to_s : Date.parse(Chronic.parse(%q(last monday)).to_s).to_s
+      d = Chronic.parse( args['-s'] || Date.today )
```

Comment:
I went ahead and submitted a commit that sets end_of_week similarly to how month sets end_of_month.